modules/ROOT/pages/resource-usage.adoc

```diff
@@ -0,0 +1,35 @@
+= View Runtime Fabric Status and Resources
```

Comment:
View or Monitor?
There is a slight difference, so we should not be using these terms interchangeably.
GLOBAL in this topic, both above this comment and below: please eval.